### ISSUE TITLE
script does not initialize

### ISSUE THREAD

**Issue by drdusandr** [CyrilCharlier/CScript#40]
script crashes when reading translation matrix (ERROR WHILST FETCHING DATA FROM SERVER), when playing on kabam.com

**Comment by CyrilCharlier** [comment]
Yes i know, i don"t know how to solve it yet but i work on it :)

**Comment by Zamaler** [comment]
I have the same problem....  no idea ???

**Comment by CyrilCharlier** [comment]
no idea yet, if somoene have an idea ...
